manual/thread_safetiness/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Thread safetiness
+layout: single
+sidebar:
+  nav: "manual"
+toc: true
+toc_sticky: true
+---
+
+ROOT classes support one of the following thread-safety levels:
+
+- Thread unsafe: objects must not be used concurrently by different threads, even if objects are not shared among threads.
```

Comment:
I think the usage of "object" here is a bit misleading: "objects must not be used concurrently..." implies _the same instance_ while "even if objects are not shared among threads" implies different per-thread instances.

Also, it's worse/more complicated than that: the paragraph mentions generic "objects" and the reader would assume we are talking about instances of one given type, but I in principle you could have race conditions between instances of different types that happen to access the same global state (and since we did not design against that nor we test for that, we have to assume this happens somewhere).

Reply:
Good point.